src/driftpy/math/amm.py

```diff
@@ -82,24 +84,19 @@ def calculate_effective_leverage(
     reserve_price: int,
     total_fee_minus_distributions: int,
 ) -> int:
+    # vAMM skew
     net_base_asset_value = (
-        quote_asset_reserve
-        - terminal_quote_asset_reserve
-        * peg_multiplier
-        // AMM_TIMES_PEG_TO_QUOTE_PRECISION_RATIO
-    )
+        (quote_asset_reserve - terminal_quote_asset_reserve) * peg_multiplier
+    ) // AMM_TIMES_PEG_TO_QUOTE_PRECISION_RATIO
 
-    local_base_asset_avlue = (
-        net_base_asset_amount
-        * reserve_price
-        // (AMM_TIMES_PEG_TO_QUOTE_PRECISION_RATIO * PRICE_PRECISION)
+    local_base_asset_value = (net_base_asset_amount * reserve_price) // (
+        AMM_TO_QUOTE_PRECISION_RATIO * PRICE_PRECISION
     )
 
-    effective_gap = max(0, local_base_asset_avlue - net_base_asset_value)
+    effective_gap = max(0, local_base_asset_value - net_base_asset_value)
 
-    effective_leverage = (
-        effective_gap // (max(0, total_fee_minus_distributions) + 1)
-        + 1 // QUOTE_PRECISION
+    effective_leverage = (effective_gap / max(0, total_fee_minus_distributions + 1)) + (
```

Comment:
there's a bug here. the 1 is added to the output of the max :

https://github.com/drift-labs/protocol-v2/blob/master/sdk/src/math/amm.ts#L517